sample-notificator-app/src/main/resources/notification-template/email-update-template.html

```diff
@@ -26,10 +26,17 @@
         </div>
       </li>
     </ul>
-    <p id="subscription-information">
-      You can also check the <a href="https://portal.qbic.uni-tuebingen.de/portal/" target="_blank">QBiC portal</a>
-      to learn more or <a id="unsubscription-link" href="" target="_blank">unsubscribe</a> from further notifications.
-    </p>
+    <div id="subscription-information">
+      <p> For more details on your samples visit the <a
+          href="https://portal.qbic.uni-tuebingen.de/portal/web/qbic/sample-tracking"
+          target="_blank">QBiC
+        portal.</a><br/>
+        To stop receiving updates on your project, you can <a id="unsubscription-link"
+                                                                  href="" target="_blank">unsubscribe</a>
+        from further
+        notifications.
```

Comment:
also here

Reply:
ah that is due to automatic formatting (to prevent lines from getting to long)